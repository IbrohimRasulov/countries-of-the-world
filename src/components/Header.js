// import { NavLink } from 'react-router-dom';

const Header = () => {
  return (
    <header className="">
      
    </header>
  );
}

export default Header;
