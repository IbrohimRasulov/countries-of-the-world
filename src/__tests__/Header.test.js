import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

it('testing rendering Header component', () => {
  const tree = renderer.create(<Router><Header /></Router>).toJSON;
  expect(tree).toMatchSnapshot();
});
