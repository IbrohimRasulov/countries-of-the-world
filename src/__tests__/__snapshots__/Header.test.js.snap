// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing rendering Header component 1`] = `[Function]`;
